common.gypi

```diff
@@ -78,12 +78,6 @@
     ##### end V8 defaults #####
 
     'conditions': [
-      ['target_arch=="arm64"', {
-        # Disabled pending https://github.com/nodejs/node/issues/23913.
-        'openssl_no_asm%': 1,
-      }, {
-        'openssl_no_asm%': 0,
```

Comment:
Does this still not need to be set (unconditionally) somewhere?

Reply:
> Does this still not need to be set (unconditionally) somewhere?

And the answer is yes it does:
https://travis-ci.com/nodejs/node/jobs/209382230#L357-L362
```console
$ ./configure
INFO: Using floating patch "tools/icu/patches/64/source/common/putil.cpp" from "tools/icu"
INFO: Using floating patch "tools/icu/patches/64/source/i18n/dtptngen.cpp" from "tools/icu"
gyp: name 'openssl_no_asm' is not defined while evaluating condition 'node_shared_openssl!="true" and openssl_no_asm==1' in /home/travis/build/nodejs/node/node.gyp while trying to load /home/travis/build/nodejs/node/node.gyp
Error running GYP
The command "./configure" exited with 1.
```

Reply:
This variable has been set unconditionally in a subsequent change.